### PR TITLE
Todo app tutorial

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: src/components/TerminalWindow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.browserWindowBody}>{children}</div>
+    </div>
+  );
+}
+
+// export const CollapsedTerminalWindow = (props: Props, sectionTitle: string) => {
```
Comment (adilansari) [inline src/components/TerminalWindow/index.tsx:42]: Will remove in next iteration

File: docs/guides/nextjs/fullstack-tutorials/todo-app/index.mdx [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+You don't have to worry about applying schema changes anymore, everything is part of development
+workflow. Learn more about this automated schema management [here](/typescript/documents/schema-bootstrap).
+:::
+
+### Run the app
+
+Let's reap the rewards. Run your Next.js app by executing `npm run dev` on terminal, you should see
+following output:
+
+// TODO: update following output
```
Comment (adilansari) [inline docs/guides/nextjs/fullstack-tutorials/todo-app/index.mdx:482]: changes incoming

File: docs/guides/nextjs/fullstack-tutorials/todo-app/index.mdx [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+them as json. This is also a good opportunity to look at how we are connecting to Tigris in
+`lib/tigris.ts`.
+
+```ts title=lib/tigris.ts
+import { DB, Tigris } from "@tigrisdata/core";
+import { Log } from "@tigrisdata/core/dist/utils/logger";
+
+const DB_NAME = "todoStarterApp";
+
+if (!process.env.TIGRIS_URI) {
```
Comment (efirs) [inline docs/guides/nextjs/fullstack-tutorials/todo-app/index.mdx:189]: This is not needed. But default it's api.preview.tigrisdata.cloud

File: sidebars.js [hunk 4]
```diff
@@ -350,7 +350,13 @@ const sidebars = {
         type: "doc",
         id: "guides/nextjs/nextjs",
       },
-      items: [],
+      collapsed: false,
+      items: [
+        {
+          type: "autogenerated",
+          dirName: "guides/nextjs/fullstack-tutorials",
```
Comment (ovaistariq) [inline sidebars.js:357]: let's have the tutorials directly under `guides/nextjs`

File: docs/guides/nextjs/fullstack-tutorials/todo-app/runDevOutput [hunk 5]
```diff
@@ -0,0 +1,25 @@
+$ ➔  npm run dev                                                                                                                                                     ±[●][main]
```
Comment (ovaistariq) [inline docs/guides/nextjs/fullstack-tutorials/todo-app/runDevOutput:1]: prepend _ to the file name

File: docs/guides/nextjs/fullstack-tutorials/todo-app/createNextAppOutput [hunk 6]
```diff
@@ -0,0 +1,37 @@
+$ ➔ npx create-next-app --example https://github.com/tigrisdata/tigris-vercel-starter/tree/main-scaffold
```
Comment (ovaistariq) [inline docs/guides/nextjs/fullstack-tutorials/todo-app/createNextAppOutput:1]: prepend _ to the file name

[the remark this answers is not in the document]
Comment (adilansari) [reply docs/guides/nextjs/fullstack-tutorials/todo-app/createNextAppOutput:1]: More like a fill in the blanks tutorial.

**Pull request merged**